config/routes.rb

```diff
@@ -164,12 +164,17 @@
         collection { get :search }
       end
 
-      resources :rules_of_origin_schemes,
-                controller: 'rules_of_origin',
-                only: %i[index]
-      get '/rules_of_origin_schemes/:heading_code/:country_code',
-          to: 'rules_of_origin#index',
-          as: :rules_of_origin
+      scope module: :rules_of_origin do
+        resources :rules_of_origin_schemes,
```

Comment:
Presumably that would enable us to skip the inflector change?

Reply:
Just tried this, I can make it work but the original path is `rules_of_origin_schemes` not `rules_of_origin/schemes` so this results in an api break - if this had been namespaced from day 1 this would definitely be the way to go but since it breaks the api I'll leave it as is I think

Reply:
Fair enough